iris_grib/_save_rules.py

```diff
@@ -670,20 +683,81 @@ def grid_definition_template_30(cube, grib):
     )
     eccodes.codes_set(grib, "LaD", cs.central_lat / _DEFAULT_DEGREES_UNITS)
     eccodes.codes_set(grib, "LoV", central_lon / _DEFAULT_DEGREES_UNITS)
+
+
+def grid_definition_template_20(cube, grib):
+    """
+    Set keys within the provided GRIB message based on
+    Grid Definition Template 3.20.
+
+    Template 3.20 is used to represent a Polar Stereographic grid.
+
+    """
+    eccodes.codes_set(grib, "gridDefinitionTemplateNumber", 20)
+
+    # Retrieve the cube coord system.
+    cs = cube.coord(dimensions=[0]).coord_system
+
+    _perspective_projection_common(cube, grib)
+
+    # Is this a north or south polar stereographic projection?
+    if cs.central_lat == -90.0:
+        centre_flag = 0x80
+    elif cs.central_lat == 90.0:
+        centre_flag = 0x0
+    else:
+        emsg = (
+            "Bipolar and symmetric polar stereo projections "
+            "are not supported in GRIB Template 3.20."
+        )
+        raise TranslationError(emsg)
+
+    if cs.true_scale_lat and cs.true_scale_lat != cs.central_lat:
+        message = (
+            f"{cs.true_scale_lat=}, {cs.central_lat=} . iris_grib can "
+            "only write a GRIB Template 3.20 file where these are identical."
+        )
+        raise TranslationError(message)
+
+    if cs.scale_factor_at_projection_origin is not None:
+        message = (
+            f"{cs.scale_factor_at_projection_origin=} . iris_grib cannot "
+            "write scale_factor_at_projection_origin to a GRIB Template 3.20 "
+            "file."
+        )
+        raise TranslationError(message)
```

Comment:
I think these 2 are basically the same case (which can be described in different ways),
and so would be better if you issued a common error message, 
e.g. "cannot save a Stereographic projection with a non-standard scale-factor".

In that case, I think you can just show all 3 of the parameters, to demonstrate why we are objecting.

Reply:
It's good information to know that these both constitute a similar problem - I didn't know that. But I worry that the user might also not know it, and so it is valuable to have the extra information about what specifically is wrong. So I have kept my original messages but added a common bit of information about these both being non-standard scale factors.

98068f48505aa91d351571b97d1cfde5fafa10cc